pyanaconda/anaconda_loggers.py

```diff
@@ -50,6 +50,7 @@ def get_main_logger():
 def get_stdout_logger():
     return logging.getLogger(constants.LOGGER_STDOUT)
 
+# TODO: this function is never used
```

Comment:
This should stay. It will be harder to implement it again in future.

Reply:
Kept.